lib/widgets/auth/auth_page.dart

```diff
@@ -65,12 +64,15 @@ class PageContents extends StatelessWidget {
               mainAxisAlignment: MainAxisAlignment.center,
               children: <Widget>[
                 PlatformSignInButton(),
-                SizedBox(
-                  height: 10,
-                ),
+                SizedBox(height: 30),
                 Row(
                   mainAxisAlignment: MainAxisAlignment.spaceEvenly,
-                  children: [EmailOptionsButton(), OtherOptionsButton()],
+                  children: [
+                    EmailOptionsButton(),
```

Comment:
added FABs for sign in options - Twitter and Facebook are disabled and greyed out for now

Reply:
sweet, theyll be bouncin soon 🏀